mindmeld/models/tagger_models.py

```diff
@@ -324,78 +357,91 @@ def evaluate(self, examples, labels):
         model_eval = EntityModelEvaluation(config, evaluations)
         return model_eval
 
-    def _get_model_constructor(self):
-        """Returns the python class of the actual underlying model"""
-        classifier_type = self.config.model_settings["classifier_type"]
-        try:
-            if classifier_type == LSTM_TYPE and LstmModel is None:
-                msg = (
-                    "{}: Classifier type {!r} dependencies not found. Install the "
-                    "mindmeld[tensorflow] extra to use this classifier type."
-                )
-                raise ValueError(msg.format(self.__class__.__name__, classifier_type))
-            return {
-                MEMM_TYPE: MemmModel,
-                CRF_TYPE: ConditionalRandomFields,
-                LSTM_TYPE: LstmModel,
-            }[classifier_type]
-        except KeyError as e:
-            msg = "{}: Classifier type {!r} not recognized"
-            raise ValueError(msg.format(self.__class__.__name__, classifier_type)) from e
+    def _dump(self, path):
 
-    def dump(self, path, config):
-        """
-        Dumps the model using joblib for the config and call's the underlying model
-        to dump its state.
+        # In TaggerModel, unlike TextModel, two dumps happen,
+        # one, the underneath classifier and two, the model metadata
 
-        Args:
-            path (str): The path to dump the model to
-            config (dict): The config containing the model configuration
-        """
-        self._clf.dump(path, config)
+        os.makedirs(os.path.dirname(path), exist_ok=True)
+        metadata = {"serializable": self._clf.is_serializable}
```

Comment:
```suggestion
        tagger_data = {"serializable": self._clf.is_serializable}
```
Since you're removing metadata altogether, this would be better aligned I think?

Reply:
We are still including the class instance itself in the dictionary along with some bunch of data related to tagging, so I'm not sure if just calling it tagger_data is meaningful 🤔